examples/esp_netif/multiple_netifs/main/multi_netif_main.c

```diff
@@ -69,13 +69,15 @@ void app_main(void)
 
     // all interfaces
     iface_info_t *ifaces[] = {
-        eth_init(ETH_PRIO),
-        wifi_init(WIFI_PRIO),
-        ppp_init(PPP_PRIO),
+        example_eth_init(ETH_PRIO),
+        example_wifi_init(WIFI_PRIO),
+        example_ppp_init(PPP_PRIO),
     };
     size_t num_of_ifaces = sizeof(ifaces) / sizeof(ifaces[0]);
 
     while (true) {
+        // For demonstration purposes we clear DNS table every iteration to exercise
```

Comment:
Could we try to move this out of a loop and have it set twice only? One with the regular expected way users would do and after that we clean and do as demo? We could consider keep the loop as is as long as we have once the expected way clearly stated.